src/main/java/core/mate/academy/model/ExcavatorProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer {
+    @Override
+    public List<Object> get() {
```

Comment:
The return type of the `get()` method should be List<Excavator>, not List<Object>.